requirements/datatype_image.txt

```diff
@@ -5,3 +5,5 @@ Pillow>=7.2
 kornia>=0.5.1
 matplotlib
 pycocotools>=2.0.2 ; python_version >= "3.7"
+git+https://github.com/pystiche/pystiche@releases/v0.7.2
+# pystiche>=0.7.2
```

Comment:
I'll push a new release as soon as we are sure nothing needs to be fixed / added to `pystiche` anymore.